libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -871,6 +900,10 @@ void AP_Baro::update(void)
             if (sensors[i].type == BARO_TYPE_AIR) {
 #if HAL_BARO_WIND_COMP_ENABLED
                 corrected_pressure -= wind_pressure_correction(i);
+#endif
+#if AP_BARO_THST_COMP_ENABLED
+                corrected_pressure -= thrust_pressure_correction(i);
```

Comment:
It would be nice to make the effect of the scale parameter explicit, in terms of what direction it does the correction.
I see you subtract the correction here probably to keep form with the lines above, but that doesn't necessarily make it clear to the one who sets the parameter.

Reply:
Added to description of param